macosx/make-app-bundle.sh

```diff
@@ -42,6 +42,15 @@ cp -r "$install/Plugins" "$install/$APPNAME.app/Contents/MacOS/"
 macdeployqt "$install/$APPNAME.app" -libpath="$install/Library"
 #sh "$dir/install-fail-tool" "$install/$APPNAME.app/Contents/Frameworks"
 
+
```

Comment:
Actually I think it's weird that this library even is a dependency. It shows up in the neuralnet-tracker.dylib. But it also shows up in the opentrack-wrapper.exe.so which was compiled by wineg++  with an absolute path /opt/local/lib/libundwind.1.dylib . That can't be right. No one will have that. I read things that it will be picked up and linked more or less by accident https://trac.macports.org/ticket/66250. not sure if it has something to do with that. In order to make opentrack-wrapper.exe.so work for others  I had to replace that reference via install_name_tool

Reply:
Actually MacPorts fixed it recently :-) https://trac.macports.org/ticket/71052 . No more absolute references to libunwind in the build :-) I'll update the commit/PR